src/Presenters/IncidentPresenter.php

```diff
@@ -73,6 +73,50 @@ public function created_at_iso()
     }
 
     /**
+     * Present diff for humans date time.
+     *
+     * @return string
+     */
+    public function scheduled_at_diff()
```

Comment:
Yeah I used to use snake case for for the views since the database fields are normally like `user_id` or `closed_at` so everything looks the same when using presenters. But lately I'm leaning to use camel case, I still have no preference lol.